### ISSUE TITLE
Not working with 1.18 game version

### ISSUE THREAD

**Issue by alemag88** [Electronics/F1NameChanger#15]
I'm aware the game only got update few hours ago but this amazing tool unfortunately is not working with patch 1.18.
Hope it will be fixed soon, thanks

**Comment by alemag88** [comment]
Ok so I'm gonna leave this here as it might help people thinking that the name changer won't work with patch 1.18.
You will need to delete the offset.json file in your name changer folder and then start the game and then search for new offsets the tool will then work fine as before.

Thanks once again to the creator of this amazing tool.

**Comment by BBence3450** [comment]
That's my problem, too. I did what you wrote but it doesn't work.

**Comment by BBence3450** [comment]
How to find a new offset?

**Comment by BBence3450** [comment]
> Oké, ezt itt hagyom, mert segíthet az embereknek abban gondolkodni, hogy a névváltó nem fog működni az 1.18-as javítással.
> Törölnie kell az offset.json fájlt a névváltó mappában, majd el kell indítania a játékot, majd új eltolások után kell kutatnia, ezután az eszköz rendben működik, mint korábban.
> 
> Még egyszer köszönöm ennek a csodálatos eszköznek a készítőjét.



> Ok so I'm gonna leave this here as it might help people thinking that the name changer won't work with patch 1.18.
> You will need to delete the offset.json file in your name changer folder and then start the game and then search for new offsets the tool will then work fine as before.
> 
> Thanks once again to the creator of this amazing tool.

How to find a new offset?

**Comment by Electronics** [comment]
I haven't had time to properly look into this new update of the game. But it sounds like just re-finding the offsets works?

In which case go to `Game -> Find Offsets` to manually trigger a new offset search.

**Comment by alemag88** [comment]
Yeah,that definitely works even after the new patch so I think this issue can be closed now.

Bence3450 probably didn't know how to look for offset but it looks like he managed to fix it too now that you explained how to do it.

I only still have some drivers showed with all capital letters in the career standing only.
Apart from that it's all perfect, thanks

**Comment by Adrii98** [comment]
Hi when i search for new offsets i am having this problem :
![image](https://user-images.githubusercontent.com/50928540/117048842-b81fae80-ad13-11eb-9577-2a556c73b576.png)
ALso i tryied the solutions proposed but it didnt work

**Comment by Electronics** [comment]
> 
> 
> Hi when i search for new offsets i am having this problem :
> ![image](https://user-images.githubusercontent.com/50928540/117048842-b81fae80-ad13-11eb-9577-2a556c73b576.png)
> ALso i tryied the solutions proposed but it didnt work

What version of the game are you running? DX12 but what language? (and version if you find that from the steam properties / in game)

**Comment by Adrii98** [comment]
1.18, language spanish

**Comment by iliager007** [comment]
Same problem, Russian language

**Comment by BaggageMan1** [comment]
Hey mate I am experiencing the same problem... DX12, English, ver. 1.18... I have tried pretty much everything i have seen here and on racedepartment to find offsets, but I am still getting the same error that @Adrii98 is getting

**Comment by Nolas91** [comment]
Hello to you all and thanks for your hard work.
Amazing job, probably the greatest mod, till today for the F1 2020. But unfortunetaly i have a question and a problem,               
Why after applying the mod and everything seems alright, my saved team mod crashes the game ?

**Comment by alemag88** [comment]
Hello @Electronics any news on a new version of this fantastic tool please?
Also will it made for F1 2021 as well?
Thanks in advance

**Comment by PsyneX** [comment]
hey, i got the same issues, try deleting the offset file, still didnt work. any help?

**Comment by werrer612** [comment]
i have the same issues, how do we fix it?

**Comment by sprChronos** [comment]
Yep i just try to play F1 2020 again, after 2021 comeout, and the name changer is error, same issues like Adrii and I play the latest update with English langugage
